### PR TITLE
chore(ibm-common): add mandatory IBM instrumentation tags

### PR DESCRIPTION
**Pull request by jeffchew**
Adding mandatory IBM intrumentation tags for Carbon Svelte.

### CODE REVIEW

**Comment by jeffchew** [comment]
@metonym fyi, this tag and footer needs to be added due to a recent IBM OPRT audit. This needs to be live by mid-August.

File: docs/src/global.css [hunk 1]
```diff
@@ -35,6 +35,11 @@ html[theme="g90"] .code-override {
   overflow-x: auto;
 }
 
+/* Override to accommodate mandatory footer */
+.bx--header~.bx--side-nav {
+  height: calc(100% - 120px);
```
Comment (metonym) [inline docs/src/global.css:40]: The magic number may need some adjusting, as the side nav appears to be cut off.

Screenshot 2024-07-17 at 8 45 21 AM

Comment (metonym) [reply docs/src/global.css:40]: Screenshot 2024-07-17 at 8 46 26 AM

**Review by metonym — CHANGES_REQUESTED** [review]
Appreciate the heads up – reviewed.

**Review by metonym — COMMENTED** [review]
@jeffchew I'm viewing this locally on Chrome – weirdly, the Sidenav height doesn't seem to fill in the viewport height until the footer is in view.

https://github.com/user-attachments/assets/13ef7d43-2281-4d2f-8ef5-922834d13191

**Comment by jeffchew** [comment]
The sidenav is fixed position and I think it probably won't be able to fill the viewport height without a layout rewrite which is what I was trying to avoid.

**Comment by metonym** [comment]
Gotcha – I'll take a closer look at this.

**Comment by jeffchew** [comment]
@metonym wondering if we can merge this for now? We need to complete the compliance remediation by the 15th.

**Comment by metonym** [comment]
Moved to #1988 

I built upon this PR so that the footer does not obstruct the side nav.

Screenshot 2024-08-07 at 8 07 42 AM

**Comment by jeffchew** [comment]
> Moved to #1988
> 
> I built upon this PR so that the footer does not obstruct the side nav.
> 

Thank you so much @metonym !

**Pull request closed without merging**